src/components/Buttons/FutureButton/FutureButtonStyle.module.css

```diff
@@ -1,39 +1,63 @@
 /* Set up some variables */
 .futureButton {
-	--bg: var(--background-accent-1-gradient);
-
+	--bg: linear-gradient(
+		90deg,
+		rgba(82, 87, 255, 0.25) 0%,
+		rgba(82, 203, 255, 0.25) 100%
+	);
 	--box-shadow: rgba(0, 209, 255, 0.1);
 	--box-shadow-heavy: rgba(0, 209, 255, 0.5);
+	--border-color: #3ca1ff;
+	--border-color-hover: #fff;
+}
+
+.futureButtonUnActive {
```

Comment:
Minor: same deal w UnActive vs Inactive - not a huge issue

Reply:
Sweet, grammar have been changed and notes were taken.